contributed_definitions/NXcalibration.nxdl.xml

```diff
@@ -79,6 +79,16 @@
              Has the calibration been applied?
         </doc>
     </field>
+    <field name="software">
```

Comment:
I'm thinking maybe we could have more expressive name. Something like `calibration_software` or `calibration_application`. It's just an idea and I would also be fine with just calling it `software`.

Reply:
I like `calibration_software`, my experience is that people mean different things by application (could also be the application of a specific calibration procedure) and it is sometime confusing in NeXus.

Will rename to `calibration_software`.